### PR TITLE
Whole docs renaming to Lotus

### PR DESCRIPTION
**Pull request by JustFragger**
I renamed all the files in the docs folder from Bitcoin-ABC to Lotus, but some still need modifications that I can't make because they are for developers :) I also added some logos to repo to make it "Lotus" in doc/logo.
**Here i made list what still needs changes:**

In ```CONTRIBUTING.md``` from line 84.
in ```DISCLOSURE_POLICY.md``` "need change PGP keys".
In doc/ ```backporting.md``` from line 9, "Identifying commits"
In doc/ ```gitian-building.md``` from line 49 "MacOS code signing" need review to end.
In doc/ ```gitian-signing.md``` from line 24 "need change SHA256_SUMS of built packages"
In doc/ ```README.md``` from line 10 "Verify"
In doc/ ```release-notes.md``` "Release notes when new release comes out"

### CODE REVIEW

**Comment by schancel** [comment]
Oh wow, thank you for doing this. I didn't see it as I get too many notifications. We have a dev chat at https://t.me/buildlotus if you are interested in joining.

File: DISCLOSURE_POLICY.md [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
 ## Responsible Disclosure Guidelines
 
-Do not disclose any bug or vulnerability on public forums, message boards, mailing lists, etc. prior to responsibly disclosing to Bitcoin ABC and giving sufficient time for the issue to be fixed and deployed.
+Do not disclose any bug or vulnerability on public forums, message boards, mailing lists, etc. prior to responsibly disclosing to Lotus and giving sufficient time for the issue to be fixed and deployed.
 Do not execute on or exploit any vulnerability.  This includes testnet, as both mainnet and testnet exploits are effectively public disclosure.  Regtest mode may be used to test bugs locally.
 
 ## Reporting a Bug or Vulnerability
 
-When reporting a bug or vulnerability, please provide the following to security@bitcoinabc.org:
+When reporting a bug or vulnerability, please provide the following to security@lotus.org:
```
Comment (schancel) [inline DISCLOSURE_POLICY.md:14]: This doesn't exist, but we can setup an email on `givelotus.org`

File: DISCLOSURE_POLICY.md [hunk 2]
```diff
@@ -47,45 +47,45 @@ gpg --full-generate-key
...
 
 ## Backup PGP Keys
 
-These PGP fingerprints and emails are provided only as backups in case you are unable to contact Bitcoin ABC via the security email above.
+These PGP fingerprints and emails are provided only as backups in case you are unable to contact Lotus via the security email above.
 
 #### Amaury Sechet
 ```
-Bitcoin ABC Lead Developer
+Bitcoin ABC Lead Developer   #<----- change devs
```
Comment (schancel) [inline DISCLOSURE_POLICY.md:59]: Yes. @EyeOfPython

File: doc/README.md [hunk 3]
```diff
@@ -59,17 +59,17 @@ Unpack the files into a directory, and then run lotus-qt.exe.
...
 
-Drag bitcoin-abc to your applications folder, and then run bitcoin-abc.
+Drag lotus to your applications folder, and then run lotus.
 
 ### Need Help?
 
 * See the documentation at the [Bitcoin Wiki](https://en.bitcoin.it/wiki/Main_Page)
 for help and more information.
-* Ask for help on the [Bitcoin ABC Subreddit](https://www.reddit.com/r/BitcoinABC/).
+* Ask for help on the [Lotus Subreddit](https://www.reddit.com/r/lotusproject/).
```
Comment (schancel) [inline doc/README.md:68]: `r/givelotus/` will become official.

File: doc/descriptors.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
-# Support for Output Descriptors in Bitcoin ABC
+# Support for Output Descriptors in Lotus Core
 
-Since Bitcoin ABC v0.20.7, there is support for Output Descriptors. This is a
+Since Lotus Core v0.20.7, there is support for Output Descriptors. This is a
```
Comment (schancel) [inline doc/descriptors.md:3]: Probably need version update here.

File: DISCLOSURE_POLICY.md (hunk 1 above)
Comment (JustFragger) [reply DISCLOSURE_POLICY.md:14]: Yep, didn't know what to put here but better change to non existing email so people will not write letters about Lotus to BitcoinABC😁

**Comment by schancel** [comment]
Accidental close. Will reopen myself.

**Pull request closed without merging**